### PR TITLE
Fix: File upload is not working in Form widget (#21341) [ED-9793]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: includes/utils.php [hunk 1]
```diff
@@ -814,6 +814,12 @@ public static function get_super_global_value( $super_global, $key ) {
 		}
 
 		if ( $_FILES === $super_global ) {
+			if ( is_array( $super_global[ $key ]['name'] ) ) {
+				$super_global[ $key ]['name'] = array_map( 'sanitize_file_name', $super_global[ $key ]['name'] );
+
+				return $super_global[ $key ];
+			}
```
Comment (StyleShit) [inline includes/utils.php:821]: why do we modify a global value instead of copying it?

File: tests/phpunit/elementor/includes/test-utils.php [hunk 2]
```diff
@@ -277,4 +277,58 @@ public function test_get_super_global_value__files() {
...
+				123,
+				123,
+			],
+		];
+
+		$sanitized = [
+			'name' => [
+				'2ffile_upload_test.php',
+				'2ffile_upload_test2.php'			],
+			'type' => [
```
Comment (StyleShit) [inline tests/phpunit/elementor/includes/test-utils.php:310]: lint

File: tests/phpunit/elementor/includes/test-utils.php [hunk 3]
```diff
@@ -277,4 +277,58 @@ public function test_get_super_global_value__files() {
...
+				123,
+				123,
+			],
+		];
+
+		// Act
+		$file = Utils::get_super_global_value( $_FILES,  'file' );
+
+		// Assert
+		$this->assertEquals( $sanitized, $file);
```
Comment (StyleShit) [inline tests/phpunit/elementor/includes/test-utils.php:332]:
```suggestion
		$this->assertEquals( $sanitized, $file );
```

File: tests/phpunit/elementor/includes/test-utils.php [hunk 4]
```diff
@@ -277,4 +277,58 @@ public function test_get_super_global_value__files() {
 		// Assert
 		$this->assertEquals( $sanitized, $file);
 	}
+
+	public function test_get_super_global_value__files_when_each_key_is_an_array() {
+		// Arrange
+		$_FILES['file'] = [
```
Comment (StyleShit) [inline tests/phpunit/elementor/includes/test-utils.php:283]: should it be cleaned up after the test?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: includes/utils.php (hunk 1 above)
Comment (Nevoss) [reply includes/utils.php:821]: As it was before

File: tests/phpunit/elementor/includes/test-utils.php (hunk 2 above)
Comment (Nevoss) [reply tests/phpunit/elementor/includes/test-utils.php:310]: Done

File: tests/phpunit/elementor/includes/test-utils.php (hunk 3 above)
Comment (Nevoss) [reply tests/phpunit/elementor/includes/test-utils.php:332]: Done

File: tests/phpunit/elementor/includes/test-utils.php (hunk 4 above)
Comment (Nevoss) [reply tests/phpunit/elementor/includes/test-utils.php:291]: Done

**Pull request closed without merging**